seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -829,7 +829,7 @@
       "description": [
         "You can add images to your website by using the <code>img</code> element, and point to a specific image's URL using the <code>src</code> attribute.",
         "An example of this would be:",
-        "<code>&#60img src=\"http://www.your-image-source.com/your-image.jpg\"&#62</code>",
+        "<code>&#60img src=\"https://www.your-image-source.com/your-image.jpg\"&#62</code>",
```

Comment:
On line **838** `http://fonts.googleapis` must be `https://fonts.googleapis` too.